src/area-chart/elements/highlighted-point.tsx

```diff
@@ -5,7 +5,7 @@ import React, { forwardRef, memo } from 'react';
 import HighlightedPoint from '../../internal/components/cartesian-chart/highlighted-point';
 
 import { ChartModel } from '../model';
-import { useSelector } from '../model/async-store';
+import { useSelector } from '../async-store';
```

Comment:
Weird to see moves inside chart folder in a commit named "Group reusable **table** elements", but okay